### PR TITLE
Profile 3.1A LB Tariffs

### PR DESCRIPTION
**Pull request by francescpuig7**
From #112 
Closes #112

### CODE REVIEW

File: enerdata/contracts/tariff.py [hunk 1]
```diff
@@ -394,16 +394,29 @@ def __init__(self):
 
 class T31A(T30A):
     """
-    3.1A Tariff
+    3.1A Tariff patched with 3.1ALB
     """
-    def __init__(self):
+    def __init__(self, **kwargs):
```
Comment (XaviTorello) [inline enerdata/contracts/tariff.py]: I think that in this case will be safer if we just define the optional parameter `kva`.

Probably will be simpler and safer, just review if `kva` is provided, if it, assign it to this tariff activating the `self.LB` flag.

What do you think @francescpuig7?

Comment (francescpuig7) [reply enerdata/contracts/tariff.py]: Done via: 0e7f4403dc010b2bb55b0d8453463077a809301d

File: spec/profiles/profile_spec.py [hunk 2]
```diff
@@ -555,6 +556,23 @@ def get_range_error():
...
+            'P3': 60,
+            'P5': 15,
+            'P6': 15
+        }
+        periods = the_tariff.energy_periods
+        total_expected = sum(balance.values())
+
+        estimation = self.profile.estimate(the_tariff, balance)
+        total_estimated = sum([x.measure for x in estimation.measures])
+        assert total_expected != total_estimated, "3.1A_LB"
```
Comment (XaviTorello) [inline spec/profiles/profile_spec.py]: Maybe here you could assert the expected amount of profiled energy

File: enerdata/contracts/tariff.py [hunk 3]
```diff
@@ -394,16 +394,30 @@ def __init__(self):
...
         self.code = '3.1A'
         self.cof = 'C'
         self.min_power = 1
         self.max_power = 450
+        self.losses = 0.04
         self.type = 'AT'
         self.require_powers_above_min_power = False
+        if kva:
+            self.LB = True
+            self.kva = kva
```
Comment (XaviTorello) [inline enerdata/contracts/tariff.py]: Remember to assert the expected kva type/s :)

File: enerdata/contracts/tariff.py [hunk 4]
```diff
@@ -394,16 +394,32 @@ def __init__(self):
...
+        self.losses = 0.04
         self.type = 'AT'
         self.require_powers_above_min_power = False
+        if kva:
+            if not isinstance(kva, int or float):
+                raise ValueError('kva must be an enter value')
+            self.LB = True
+            self.kva = kva
+        else:
+            self.LB = False
```
Comment (ecarreras) [inline enerdata/contracts/tariff.py]: I don't like the properties in upper cases, in a standard it denotes CONSTANT_VALUES and it isn't a constant it's a property. What about `low_voltage_measure` or something

Comment (francescpuig7) [reply enerdata/contracts/tariff.py]: I think same!

Comment (francescpuig7) [reply enerdata/contracts/tariff.py]: Done via: https://github.com/gisce/enerdata/pull/112/commits/5a4796099ab50f0630fdc415c7436f3bcace0191

File: enerdata/contracts/tariff.py (hunk 3 above)
Comment (francescpuig7) [reply enerdata/contracts/tariff.py]: Doned

File: spec/profiles/profile_spec.py (hunk 2 above)
Comment (francescpuig7) [reply spec/profiles/profile_spec.py]: Doned

**Pull request merged**